.npmignore

```diff
@@ -0,0 +1,4 @@
+*
+!dist/**
+!package.json
+!README.md
```

Comment:
i added this here, when publishing the package, only the dist and package.json folder will be included - this can be used to import the `core-contracts` into other projects.